src/ui/A/A.jsx

```diff
@@ -58,13 +59,24 @@ function A({ to, hook, variant = 'default', children, isExternal, ...props }) {
     'inline-flex items-center gap-1'
   )
 
+  const handleClick = () => {
```

Comment:
Should we grab the event here and pass it along to the `onClick` call handler in case we're using it anywhere?